cluster/src/main/java/org/apache/iotdb/cluster/log/manage/serializable/LogDequeSerializer.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.cluster.log.manage.serializable;
+
+import java.util.List;
+import org.apache.iotdb.cluster.log.Log;
+
+public interface LogDequeSerializer {
+
```

Comment:
I want to add a new function “void append(List<Log> entries)” to support batch append so that it can be easily embedded with new design.You can do it without thinking about efficiency, and you can rewrite it after I've merged it. Of course, it would be better to focus on efficiency in the first place.

Reply:
I would like to add another function "void removeCompactedEntries(long index)" which means the persisted logs which index prior to index is able to be deleted.Of course, you can delete it lazily.But as we are not clear when to delete persisted raft logs so far,so you can implement it later.Just for reminding~

Reply:
Thanks~, I will add  “void append(List entries)” and removeCompactedEntries can be achieved by remove function. In LogDequeSerializer, we don't know index but only the location of the logs. So it's memory log manager's responsibility to use index to find how many log we should remove.